### PR TITLE
Display/resize multiple avatar icons for group messages

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - Add support for displaying multiple icons

## Screenshots

Updated:
Screenshot 2023-11-09 at 11 23 39 AM

### CODE REVIEW

File: src/screens/MessageScreen.tsx
Comment (sternetj) [inline src/screens/MessageScreen.tsx:323]: Any chance we could make it round with a background, similar to iOS?
IMG_4065

File: src/screens/MessageScreen.tsx
Comment (sternetj) [inline src/screens/MessageScreen.tsx:237]: We should probably build this with injected style settings

File: src/screens/MessageScreen.tsx
Comment (sternetj) [inline src/screens/MessageScreen.tsx:326]: I think if we throw a justify center on here it would look a bit better when there are just three. I think it would look like this
image

**Review by sternetj — COMMENTED** [review]
Thanks for doing this I think this will be a really nice enhancement. Couple of suggestions.

**Comment by markdlv** [comment]
This is looking great!  I'll wait for the feedback items, but LGTM otherwise.

**Review by sternetj — APPROVED** [review]
Looks awesome!

**Pull request merged**
